controller/identity.go

```diff
@@ -25,11 +26,31 @@ func NewIdentityController(service *goa.Service, db application.DB) *IdentityCon
 // List runs the list action.
 func (c *IdentityController) List(ctx *app.ListIdentityContext) error {
 	return application.Transactional(c.db, func(appl application.Application) error {
-		result, err := appl.Identities().List(ctx.Context)
+		modelIdentities, err := appl.Identities().List(ctx.Context)
 		if err != nil {
 			jerrors, _ := jsonapi.ErrorToJSONAPIErrors(goa.ErrInternal(fmt.Sprintf("Error listing identities: %s", err.Error())))
 			return ctx.InternalServerError(jerrors)
 		}
-		return ctx.OK(result)
+		appIdentities := app.IdentityArray{}
+		appIdentities.Data = make([]*app.IdentityData, len(modelIdentities))
+		for index, modelIdentity := range modelIdentities {
+			appIdentityData := ConvertIdentityFromModel(modelIdentity)
+			appIdentities.Data[index] = appIdentityData
```

Comment:
Chore: This kind of practice is wrong. I believe it is sometimes due to the habit of getting the value of a function like `result, err :=` and then use `result`. Here, you define a new variable to only use it below. Why not a direct assignation ?